api/src/main/java/org/openmrs/api/ConditionService.java

```diff
@@ -62,6 +62,17 @@
 	@Authorized({ PrivilegeConstants.GET_CONDITIONS })
 	List<Condition> getActiveConditions(Patient patient) throws APIException;
 
+    /**
+	 * Gets a patient's active and inactive conditions
+	 *
+	 * @param patient - the patient to retrieve conditions for
+	 * @throws APIException   
+	 * @return a list of the patient's conditions
+	 * @since 2.2
```

Comment:
Is this the current master branch version?

Reply:
No, it is the version from which these changes are required. Is it supposed to be the current version?

Reply:
Supposed to be the version of master when these changes are introduced.

Reply:
@dkayiwa  I updated the PR. Please take a look :)